repository/user_repository.py

```diff
@@ -32,15 +32,6 @@ def delete_user(self, email: str) -> None:
             self.db.delete(user)
             self.db.commit()
     
-    def assign_mentor(self, mentor_email: str, mentee_email: str) -> None:
-        mentee = self.get_user_by_email(mentee_email)
-        mentor = self.get_user_by_email(mentor_email)
-        if mentee and mentor:
-            mentee.mentor_email = mentor_email  
-            self.db.commit()
-    
-    def unassign_mentor(self, mentor_email: str) -> None:
-        mentees = self.db.query(User).filter(User.mentor_email == mentor_email).all()
-        for mentee in mentees:
-            mentee.mentor_email = None
-        self.db.commit()
+    def get_all_mentees(self, mentor_email: str) -> list[Type[User]]:
```

Comment:
Is it not supposed to be `User.mentorEmail== mentor_email`

Reply:
No